src/main/scala/IntegrationTests.scala

```diff
@@ -0,0 +1,36 @@
+package tests
+
+import org.apache.spark.SparkContext
+import pipelines.images.cifar.RandomPatchCifar
+import pipelines.images.cifar.RandomPatchCifar.RandomCifarConfig
+import pipelines.images.voc.VOCSIFTFisher
+import pipelines.images.voc.VOCSIFTFisher.SIFTFisherConfig
+import pipelines.text.NewsgroupsPipeline
+import pipelines.text.NewsgroupsPipeline.NewsgroupsConfig
+
+object IntegrationTests {
+  val cifar = RandomPatchCifar.run(_: SparkContext,
+    RandomCifarConfig(
+      trainLocation="./cifar_train.bin",
```

Comment:
For consistency I wonder if i should just copy these to hdfs anyway - they'll fit in one block and the data loader will repartition them.